planetiler-core/src/main/java/com/onthegomap/planetiler/Profile.java

```diff
@@ -157,6 +158,8 @@ default boolean isOverlay() {
     return false;
   }
 
+  default Map<String,String> extraMetadata() { return Map.of(); }
```

Comment:
I wonder if we should include `archive` in the name here to make it clear where the metadata is going?

Reply:
changed to `extraArchiveMetadata`